scripts/jest.config.js

```diff
@@ -3,6 +3,11 @@ const path = require('path');
 /** Jest config for packages within the just monorepo */
 module.exports = {
   roots: ['<rootDir>/src'],
+  // moduleDirectories: [
```

Comment:
remove or uncomment?

Reply:
oh i cleaned it up but forgot to push this change